recipes/newrelic/infrastructure/kubernetes.yml

```diff
@@ -658,8 +658,8 @@ install:
             curl -s -H "Content-Type: application/json" -d "${BODY}" "${URL}" > $KUBECTL_YAML_DIR/newrelic-k8s.yml
 
             if [[ "${NR_CLI_PIXIE}" == "true" ]]; then
-              $SUDO $KUBECTL apply -f https://raw.githubusercontent.com/pixie-io/pixie/main/k8s/operator/helm/crds/olm_crd.yaml
-              $SUDO $KUBECTL apply -f https://raw.githubusercontent.com/pixie-io/pixie/main/k8s/operator/crd/base/px.dev_viziers.yaml
+              $SUDO $KUBECTL create -f https://raw.githubusercontent.com/pixie-io/pixie/main/k8s/operator/helm/crds/olm_crd.yaml 2>/dev/null
+              $SUDO $KUBECTL create -f https://raw.githubusercontent.com/pixie-io/pixie/main/k8s/operator/crd/base/px.dev_viziers.yaml 2>/dev/null
```

Comment:
this command still returns an exit code of 1. can you double check that the install script ignores the exit status and continues to run?

additionally you might want to consider using a released version here such as https://raw.githubusercontent.com/pixie-io/pixie/release/operator/v0.1.0/k8s/operator/helm/crds/olm_crd.yaml
instead of using `main`

Reply:
only keep stderr with 2>&1 >/dev/null; if the error include any other than AlreadyExist, collect the error in the DEBUG_MESSAGE

Should we exist here?